templates/ftp_listener/src/ftp_listener/Module.md

```diff
@@ -0,0 +1,18 @@
+# FTP Listener
+
+In this template we will implement an FTP listener which will listen to a remote directory and periodically notify the 
+addition and deletion of files that comply with the specified file pattern.
+
+## How to run the template
+
+1. Alter the config file `src/ftp_listener/resources/ballerina.conf` according to your requirement. 
+
+2. Execute the following command in the project directory `ftp_listener` to run the service.
+```bash
+ballerina run ftp_listener --b7a.config.file=src/ftp_listener/resources/ballerina.conf 
```

Comment:
```suggestion
ballerina run ftp_listener --b7a.config.file=src/<module_name>/resources/ballerina.conf 
```
Since the user can add any name for the module, i think its better not to specify this template module name.

Reply:
Added.